proto/viam/app/v1/app.proto

```diff
@@ -820,6 +820,8 @@ message FragmentHistoryEntry {
   google.protobuf.Timestamp edited_on = 2 [(tagger.v1.tags) = "bson:\"edited_on\" json:\"edited_on\""];
   Fragment old = 3 [(tagger.v1.tags) = "bson:\"old\" json:\"old\""];
   AuthenticatorInfo edited_by = 4 [(tagger.v1.tags) = "bson:\"edited_by\" json:\"edited_by\""];
+  string revision = 5 [(tagger.v1.tags) = "bson:\"revision\" json:\"revision\""];
+  google.protobuf.Struct config = 6 [(tagger.v1.tags) = "bson:\"fragment\" json:\"fragment\""];
```

Comment:
double checking, this field `config` will eventually replace `old`? also, should the bson tags be `config` as well?

Reply:
Oops my bad about the tags, will update. Yes I think `old` will be deprecated. It doesn't make sense to drive the fragment version history off of `old` if we are storing that version's `config` as well. Plus its confusing to have the entire fragment in the response.

We have a lot of tasks to complete to migrate completely off of `old` and then we can deprecate it everywhere.